cardano-config/src/Cardano/Config/Orphanage.hs

```diff
@@ -50,3 +57,13 @@ instance FromJSON Update.ApplicationName where
   parseJSON invalid  =
     panic $ "Parsing of application name failed due to type mismatch. "
     <> "Encountered: " <> (Text.pack $ Prelude.show invalid)
+
+instance Serialise (HeaderHash blk) => ToJSON (Tip blk) where
+  toJSON TipGenesis = object [ "genesis" .= True ]
```

Comment:
Is this an appropriate output for genesis?

```json
{
  "genesis": true
}
```

Reply:
What about something like:

```
{
  "currentBlock": "genesis"
}
```